miden-lib/src/accounts/wallets/mod.rs

```diff
@@ -25,9 +26,20 @@ use crate::accounts::{auth::RpoFalcon512, components::basic_wallet_library};
 /// This component supports all account types.
 pub struct BasicWallet;
 
+impl BasicWallet {
+    /// Returns a reference to the Basic Wallet library whose procedures can be imported from
+    /// `account_components::basic_wallet`.
+    ///
+    /// This can be used in the assembly of programs that want to call procedures from this
+    /// component.
```

Comment:
One thing to note is that all the procedures in the basic wallet are just re-exports from `miden-lib`, and so if `miden-lib` is imported into an assembler, there is no need to import basic wallet code separately.

On the other hand, basic wallet code relies on `miden-lib`, so - importing it without importing `miden-lib` will not be sufficient.

We should probably adjust the comment to reflect the above.